src/gilded_rose.js

```diff
@@ -49,7 +49,7 @@ export function updateQuality(items) {
   const result = []
   for (var i = 0; i < items.length; i++) {
     if (items[i].name === 'Sulfuras, Hand of Ragnaros') {
-      break;
+          result.push({...items[i]})
```

Comment:
What's the purpose of the spread syntax here? If you're not overwriting any props of this object, I think you could just do `result.push(items[i])`, no?

Reply:
lol, I was trying to play it safe...... but I see that I should be able to do it that way. TY!